model/build.gradle

```
@@ -3,7 +3,13 @@ apply plugin: 'com.google.protobuf'
 
 protobuf {
   protoc {
-    artifact = 'com.google.protobuf:protoc:3.8.0'
+    // To build protoc in M1 mac. For Context, see:
+    // https://github.com/grpc/grpc-java/issues/7690.
```

```suggestion
    // To build protoc in M1 mac. For context, see: #3912.
```

Prefer linking to our issue for context.

---

Done